src/Altinn.Platform/Altinn.Platform.Storage/Storage/Controllers/InstanceEventsController.cs

```diff
@@ -58,18 +54,23 @@ public async Task<ActionResult> Post([FromBody] InstanceEvent instanceEvent)
         /// <summary>
         /// Retrieves all instance events related to given instance id, listed event types, and given time frame from instanceEvent collection.
         /// </summary>
-        /// <param name="instanceId"> Id of instance to retrieve events for. </param>
-        /// <param name="eventTypes">Array of event types to filter the events by./param>
-        /// <param name="fromDateTime"> Lower bound for DateTime span to filter events by.</param>
-        /// <param name="toDateTime"> Upper bound for DateTime span to filter events by.</param>
+        /// <param name="instanceOwnerId">instance owner id</param>
+        /// <param name="instanceGuid"> Id of instance to retrieve events for. </param>
+        /// <param name="eventTypes">Array of event types to filter the events by.</param>
+        /// <param name="from"> Lower bound for DateTime span to filter events by.</param>
+        /// <param name="to"> Upper bound for DateTime span to filter events by.</param>
         /// <returns>List of instance events.</returns>
+        /// <!--
```

Comment:
Should probably have the same format for the examples on the other methods? 
Or remove the examples all togheter. Are we able to place comments like these in swagger now?

Reply:
StyleCop complains that the xml is not correct. Therefore I have commented some of the elements.